### PR TITLE
journal-week-4

### PR DESCRIPTION
**Pull request by jeffworley**

### CODE REVIEW

**Comment by nickycadavillo** [comment]
:+1: I like where your journal is headed.  The growth/fixed mindset is something we should be continually reflecting on so your post is a good reminder.  Also, i'm looking forward to your tutorial because pseudo-code will help in the chunking/planning of our assignments.

**Comment by s4dmw** [comment]
- [x] journal - great idea to revisit an "old" topic to see how views have changed in such a short time
- [x] tutorial - looking forward to reading more about pseudo code...it's still a mysterious thing to me
- [x] :+1:

**Comment by ericksonk** [comment]
Woah can't believe we're already 1 month in! It will be interesting to see how your mindset has changed over the course of the program.

Pseudo code saved my life. Looking forward to see you explain your approach. :+1:

**Comment by jeremyrist** [comment]
:+1:

File: tutorial-week-4.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+### Tutorial Brainstorming v1.0:
```
Comment (kellymurray) [inline tutorial-week-4.md:1]: Very useful tutorial!

**Pull request merged**